torchvision/ops/boxes.py

```diff
@@ -114,7 +114,12 @@ def _batched_nms_vanilla(
 
 def remove_small_boxes(boxes: Tensor, min_size: float) -> Tensor:
     """
-    Remove boxes which contains at least one side smaller than min_size.
+    Remove every box from :attr:`boxes` which contains at least one side length
+    that is smaller than :attr:`min_size`.
+
+    .. warning::
+        For sanitizing a :class:`~tv_tensors.BoundingBoxes` object, consider using
+        the transform :func:`~torchvision.transforms.v2.SanitizeBoundingBoxes` instead.
```

Comment:
Here and in other places this should be prepended with `torchvision` for the link to work:

```suggestion
        For sanitizing a :class:`~torchvision.tv_tensors.BoundingBoxes` object, consider using
        the transform :func:`~torchvision.transforms.v2.SanitizeBoundingBoxes` instead.
```